src/store/membersSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const membersSlice = createSlice({
```

Comment:
Is there any reason to create a slice for this members data?

Reply:
To set the members data, I thought it would be better to have it separately in a slice

Reply:
We are showing members data inside a component and its now shared across anywhere in the app.
So we can go with simple state instead of setting this in global redux level.